arcgis-java-toolkit-examples/build.gradle

```diff
@@ -95,5 +93,5 @@ task productionZip(type: Zip) {
 }
 
 wrapper {
-    gradleVersion = '7.3'
+    gradleVersion = '7.5'
```

Comment:
Latest release us 7.5.1 - we should aim for that as Gradle recommend this version.